lib/delighted/enumerable_resource_collection.rb

```diff
@@ -1,4 +1,5 @@
 module Delighted
+  # Deprecating. Use +EnumerableResource+.
   class EnumerableResourceCollection < Array
```

Comment:
Keeping this `EnumerableResourceCollection < Array` for backwards compatibility, in case some users are using array specific features. The new`EnumerableResource include Enumerable` should be more flexible.